### ISSUE TITLE
✅ 🔗 🐞 Loop Builder - Apostrophe's being replaced with 8217 After E3.21 update (Duplicate of #25860)

### ISSUE THREAD

**Issue by adam-poly** [elementor/elementor#25899]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Using the Loop builder to make custom post cards, add the 'Content' widget, then turn on to 'Apply to Post Content', apostrophes will show correctly, but then when you change the Excerpt Length to anything the apostrophe will show as '8217'. See video. Tested on 2 sites using Hello Elementor. 

https://github.com/elementor/elementor/assets/163076118/33796f93-a78a-4b7e-bf07-1e057904c348

### Steps to reproduce

Make a new loop
Add in content widget
Set loop as 'Post'
Turn on 'Apply to Post Content' 
Change excerpt length to anything. 

https://github.com/elementor/elementor/assets/163076118/ca3e26c7-9708-4653-85ca-e463aef4b941

### Expected behavior

Apostrophes should not show as 8217s

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

```txt
Nginx
Wordpress 6.5.2
Elementor 3.21.3
Elementor Pro 3.21.1
```

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello @adam-poly!

Thank you for your submission! 🙏 

### 🔗 🐞 This issue is a Duplicate of #25860

Which is on the same subject. 

It has been solved with the release of Elementor Pro version 3.21.1.

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Next time, please provide the FULL Elementor System Info as required in the Bug Report form.

Kind regards

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.21.1 and v3.21.2! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
